test/integration/testMaxLobSize.js

```diff
@@ -95,6 +98,52 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true'){
       });
     });
 
+    describe('test put get max LOB size', function () {
+      before(async function () {
+        connection = testUtil.createConnection();
+        await testUtil.connectAsync(connection);
+        await testUtil.executeCmdAsync(connection, createTable);
+        await testUtil.executeCmdAsync(connection, allowLobExternalScan);
+      });
+
+      after(async () => {
+        await testUtil.executeCmdAsync(connection, removeStage);
+        await testUtil.dropTablesIgnoringErrorsAsync(connection, [tableName]);
+        await testUtil.destroyConnectionAsync(connection);
+      });
+
+      it('test - copy to stage where size > 16MB', function (done){
+        const rowData = mediumSizeTableData.C1 + ',' + mediumSizeTableData.C2 + ',' + mediumSizeTableData.C3 + '\n';
+        const tmpFile = testUtil.createTempFile(os.tmpdir(), testUtil.createRandomFileName(), rowData);
+
+        let putQuery = `PUT file://${tmpFile} ${stageName}`;
+        // Windows user contains a '~' in the path which causes an error
+        if (process.platform === 'win32') {
+          const fileName = path.basename(tmpFile);
+          putQuery = `PUT file://${process.env.USERPROFILE}\\AppData\\Local\\Temp\\${fileName} ${stageName}`;
+        }
+
+        async.series([
+          function (callback) {
+            // Upload file
+            connection.execute({
+              sqlText: putQuery,
+              complete: function (err) {
+                if (err) {
+                  callback(err);
+                }
+                callback();
+              }
+            });
+          },
+          function (callback) {
+            // Copy into temp table
+            testUtil.executeCmd(connection, copyIntoTable, callback);
+          }
+        ], done);
```

Comment:
Is it needed to verify whether the data has been inserted?

Reply:
I don't think that this is needed, we only want to execute this without any errors. But added assertion if the status is 'LOADED'

Reply:
ok